batch_import/EXAMPLE.md

```diff
@@ -0,0 +1,210 @@
+with the `route` tool:
+
+**On OSX**
+```sh
+$ route get example.com   
+route to: 93.184.216.34
+destination: default
+       mask: default
+    gateway: 192.168.1.1
+  interface: en0 ## <--- this is the interface name ##
+      flags: <UP,GATEWAY,DONE,STATIC,PRCLONING>
+ recvpipe  sendpipe  ssthresh  rtt,msec    rttvar  hopcount      mtu     expire
+       0         0         0         0         0         0      1500         0
+```
+
+**On Linux**
+```sh
+$ route
+Kernel IP routing table
+Destination     Gateway         Genmask         Flags Metric Ref    Use Iface 
+default         192.168.1.1     0.0.0.0         UG    0      0        0 enp0s3 ## <-- this is the interface name
+192.168.1.0     *               255.255.255.0   U     0      0        0 enp0s3
+``` 
+
+Then run the `arp-scan` tool (replace `eth0` with your interface name)
+```sh
+$ sudo arp-scan --interface=eth0 --localnet 
+Interface: en0, datalink type: EN10MB (Ethernet)
+Starting arp-scan 1.9 with 256 hosts (http://www.nta-monitor.com/tools/arp-scan/)
+192.168.1.1    20:e5:2a:02:f6:fc    NETGEAR INC.,
+192.168.1.10    28:10:7b:07:0c:51    D-Link International
+192.168.1.20    b8:27:eb:a0:25:bc    Raspberry Pi Foundation
+192.168.1.30    00:1e:06:33:d7:1a    WIBRAIN
+192.168.1.51    be:fe:11:27:02:88    (Unknown)
+# ....
+```
+
+Look for the entry with the MAC address of your Camio Box (for VMs this will start with `BE:FE:11`), and note down the IP address.
+
+##### Using The Fing Application
+
+Download the [Fing application](https://www.fing.io/) to your phone. Open the app and click the 'refresh' button on the top bar. Ths will kick off a scan
+of the network, displaying all of the devices that it has located. Look through the list for the MAC address of your Camio Box and note down the IP-address listed.
+
+
+##### Constructing the File-Parsing Regex
+
+For the sake of the example, let's say that your directory of video files for batch-import is located at `~/batch_videos/`, and the files have the
+format of 
+
+```
+$ ls ~/batch_videos
+CAMERA_FRONT-rand-1475973147.mp4
+CAMERA_FRONT-rand-1475973267.mp4
+CAMERA_FRONT-rand-1475973350.mp4
+```
+
+Then you would use the following string as the regular-expression passed to the video-import script.
+
+`.*/(?P<camera>\w+?)\-.*\-(?P<epoch>\d+)\.mp4`
+
+This captures the `CAMERA_FRONT` part as the camera name (used for registering the camera with Camio), and parses the `1475973147` part as the Unix-timestamp
+of the video.
+
+####  Running the video-importer Script
+
+You are now ready to batch-import videos to Camio through your Camio Box. 
+
+To start, go to the `video-importer` directory.
+
+```sh
+$ cd ~
+$ cd examples/batch_import/video-importer
+```
+
+Run the importer with all of the values we've assembled in the previous steps
+
+```bash
+$ python importer.py \
+  --regex ".*/(?P<camera>\w+?)\-.*\-(?P<epoch>\d+)\.mp4" \
+  --folder "~/batch_videos" \
+  --host 192.168.1.57 \
```

Comment:
Make this 192.168.1.51 or change 192.168.1.51 to 192.168.1.57 to be consistent with your scan example above.